system/views/admin/rulesEngine/index.cfm

```diff
@@ -5,8 +5,9 @@
 
 
 <cfoutput>
-	<div class="top-right-button-group">
+	<div class="top-right-button-group" style="margin: -8px 4px 0px 0px;">
```

Comment:
Not sure on this one. I think we need a global fix for this issue with the 'top-right-button-group'. i.e. some kind of `event.addAdminTopRightButton()`, `event.getAdminTopRightButtons()` set of methods. These could then be output at the very end of the request in a CSS layout that doesn't require absolute positioning.

Can you remove this bit from the PR and we can then merge the other change?